packages/docs/src/components/GithubLinks.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Button } from '@cmsgov/design-system';
+import classnames from 'classnames';
+
+interface GithubLinks {
+  /**
+   * Describes if the links are on a dark background
+   */
+  onDark?: boolean;
```

Comment:
I'm thinking this might be defined at the theme/css level during a swap and may not be necessary?

Reply:
I included this because this component is used both on in the desktop header (which has a dark background) and in the mobile menu (which has a light background). I needed a way to apply slightly different button styles based on those two uses.

It's not related to the whole site's theme.